scripts/build-installer-unix.sh

```diff
@@ -148,37 +148,46 @@ cd installers
         $INSTALLER/bin/load-certificate -k macos-build.keychain -f macos.p12
     fi
 
-    echo "Generating the installer.."
-    $INSTALLER/bin/make-installer
-
-    INSTALLER_PKG="Daedalus-installer-${DAEDALUS_VERSION}.pkg"
-    APP_NAME="csl-daedalus"
-
-    if test -d dist -a -f "dist/${INSTALLER_PKG}"; then
-        echo "Uploading the installer package.."
-        cd dist
-        mkdir -p ${APP_NAME}
-        mv "${INSTALLER_PKG}" "${APP_NAME}/${INSTALLER_PKG}"
-
-        if [ -n "${BUILDKITE_JOB_ID:-}" ]; then
-            export PATH=${BUILDKITE_BIN_PATH:-}:$PATH
-            buildkite-agent artifact upload "${APP_NAME}/${INSTALLER_PKG}" s3://${ARTIFACT_BUCKET} --job $BUILDKITE_JOB_ID
-        elif test -n "${TRAVIS_JOB_ID:-}" -a -n "${upload_s3}"
-        then
-            echo "$0: --upload-s3 passed, will upload the installer to S3";
-            retry 5 $nix_shell -p awscli --run "aws s3 cp '${APP_NAME}/${INSTALLER_PKG}' s3://daedalus-internal/ --acl public-read"
-        fi
-        cd ..
-    else
-        echo "Installer was not made."
-        mkdir -p dist/${APP_NAME}
-    fi
-
-    if test -n "${test_install}"
-    then echo "$0:  --test-install passed, will test the installer for installability";
-         case ${OS_NAME} in
-                 darwin ) sudo installer -dumplog -verbose -target / -pkg "dist/Daedalus-installer-${DAEDALUS_VERSION}.pkg";;
-                 linux )  echo "WARNING: installation testing not implemented on Linux" >&2;; esac; fi
+    case ${OS_NAME} in
```

Comment:
`make-installer` already conditionalizes per OS

Reply:
@domenkozar, so you suggest moving installer testing into the installer generator?

Reply:
Sure - why not it really just works for darwin.

Reply:
Done

Reply:
@domenkozar, it's still referenced -- for the Cardano download.